content/cpp/concepts/math-functions/terms/nan/nan.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: 'nan()'
+Description: 'The nan() function in C++ returns a quiet NaN (Not-A-Number) value of type double.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+## Syntax
+
+```pseudo
+nan(x)
+```
+
+## Return value
+
+The nan() function returns quiet NaN value.
+
+## Example
+
+The following example uses the `nan()` function to find the quiet NaN value :
+
+```cpp
```

Comment:
This example will not run as written. `src` is not defined and hex should be std::hex. Please revise it.

Reply:
@chiragcode11, the code will not work. Please update this with a working example.